pkg/email/system_intake_close_test.go

```diff
@@ -0,0 +1,84 @@
+package email
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"golang.org/x/net/context"
+
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+func (s *EmailTestSuite) TestCloseIntakeRequestNotification() {
+	ctx := context.Background()
+	intakeID := uuid.MustParse("24dd7736-e4c2-4f67-8844-51187de49069")
+	requestName := "Hotdog/Not Hotdog Program"
+	requester := "Dr Fishopolis"
+	submittedAt := time.Now()
+	requestLink := "http://localhost:3000/governance-task-list/" + intakeID.String()
```

Comment:
It doesn't matter too much (the tests pass locally and in CI), but rather than hardcoding `http://localhost:3000`, you can get the URL scheme and host from `s.config`. Example from another test file:
```go
trbAdminAdviceLetterLink := fmt.Sprintf(
		"%s://%s/%s",
		s.config.URLScheme,
		s.config.URLHost,
		path.Join("trb", trbID.String(), "advice"),
	)
```

Reply:
Oh OK, good call out.